packages/itwinui-css/src/button/base.scss

```diff
@@ -41,7 +41,7 @@
   svg {
     @include mixins.iui-icon-style('m');
     transition: fill var(--iui-duration-1) ease-out;
-    fill: var(--_iui-button-icon-fill, currentColor);
+    fill: var(--_iui-button-icon-fill, var(--iui-color-icon));
```

Comment:
Wouldn't it be safer that the icon is the same color as the text? Also, although the difference between `currentColor` is not apparent in the default case, it is apparent in the cta, high-visibility, and disabled cases.

https://github.com/iTwin/iTwinUI/assets/45748283/9e8e3c2d-8a81-4d74-8e2d-33eeac4f7a52

Reply:
Reverted back to `currentColor` because it is a safer fallback and if we did our jobs right it should never be applied anyways.